src/ledger/test/LedgerTxnTests.cpp

```diff
@@ -2804,6 +2819,308 @@ TEST_CASE("Erase performance benchmark", "[!hide][erasebench]")
 #endif
 }
 
+TEST_CASE("LegerTxnRoot prefetch soroban entries", "[ledgertxn]")
+{
+    auto ensureLedgerKeyMeterCanLoad =
```

Comment:
I don't think this function is necessary in this test, as we have a dedicated test case specifically for ledger key meter. I think this test should just focus on whether the keys are actually loaded or not.